package.json

```diff
@@ -70,6 +70,7 @@
     "posttest": "rm -rf cli-tests-*",
     "postunit": "npm run posttest",
     "t": "node bin/run.js test -J -sfails.txt",
+    "prepare": "node bin/dump-taprc-schema.js > docs/static/taprc.schema.json",
```

Comment:
This isn't going to get run on docs generation, because `gatsby` is executed in the docs folder.

It would probably make the most sense in `docs/package.json` as a `prebuild` command.

Reply:
Ok I will change that.